modules/quotas-requiring-explicit-quota.adoc

```diff
@@ -8,44 +8,100 @@
+    name: storage-consumption
+  spec:
+    hard:
+      persistentvolumeclaims: "10" <1>
+      requests.storage: "50Gi" <2>
+      gold.storageclass.storage.k8s.io/requests.storage: "10Gi" <3>
+      silver.storageclass.storage.k8s.io/requests.storage: "20Gi" <4>
+      silver.storageclass.storage.k8s.io/persistentvolumeclaims: "5" <5>
+      bronze.storageclass.storage.k8s.io/requests.storage: "0" <6>
+      bronze.storageclass.storage.k8s.io/persistentvolumeclaims: "0" <7>
+----
+<1> The total number of persistent volume claims in a project.
+<2> Across all persistent volume claims in a project, the sum of storage requested cannot exceed this value.
+<3> Across all persistent volume claims in a project, the sum of storage requested in the gold storage class cannot exceed this value.
+<4> Across all persistent volume claims in a project, the sum of storage requested in the silver storage class cannot exceed this value.
+<5> Across all persistent volume claims in a project, the total number of claims in the silver storage class cannot exceed this value.
+<6> Across all persistent volume claims in a project, the sum of storage requested in the bronze storage class cannot exceed this value. When this is set to 0, it means bronze storage class cannot request storage.
+<7> Across all persistent volume claims in a project, the sum of storage requested in the bronze storage class cannot exceed this value. When this is set to 0, it means bronze storage class cannot create claims.
+
+. Create a new project request template from the modified `template.yaml` file, in the `openshift-config` namespace. By default, the template will be called `project-request`:
++
+----
+$ oc create -f template.yaml -n openshift-config
+----
+
+. Reference the new project request template in the cluster's project configuration resource:
+.. Access the project configuration resource for editing:
++
+** Using the web console:
+... Navigate to the *Administration* -> *Cluster Settings* page.
+... Click *Global Configuration* to view all configuration resources.
+... Find the entry for *Project* and click *Edit YAML*.
++
+** Using the CLI:
+... Edit the `project.config.openshift.io/cluster` resource:
++
+----
+$ oc edit project.config.openshift.io/cluster
+----
++
+.. Update the `spec` section of the project configuration resource to include the `projectRequestTemplate` and `name` parameters. The following example references the default project request template name `project-request`:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Project
+metadata:
+  ...
+spec:
+  projectRequestTemplate:
+    name: project-request
+----
+
+. Verify that the resource quota is applied when new projects are created:
+.. Create a new project:
++
+----
+oc new-project my-project
+----
++
+.. Verify that the resource quota has been applied. The following example describes the `storage-consumption` resource quota:
++
+----
+oc describe resourcequotas storage-consumption
```

Comment:
$ oc

Same as the last command, I'd make the resource quota a variable and explain how you get the name.

Reply:
Resolved.